packages/scikit-learn/index.rst

```diff
@@ -395,7 +391,7 @@ object API and several additional features, including smart initialization.)
 
         - Raw image
 
-        - K-means quantization
+        - K-means quantization (K=5)
```

Comment:
On the image, on a quick glance it was unclear for a non-connoisseur if K in *K-means* is a *parameter* or the *name* of the method (I hope adding `K=5` helps).